docs/admin/upgrade.md

```diff
@@ -2,4 +2,28 @@
 
 ## Upgrade Guide
 
-When a new release comes out it may be necessary to run a migration of the database to account for any changes in the data models used by this plugin. Execute the command `nautobot-server post-upgrade` within the runtime environment of your Nautobot installation after updating the `nautobot-ssot` package via `pip`.
+When a new release comes out it may be necessary to run a migration of the database to account for any changes in the data models used by this Nautobot app. Execute the command `nautobot-server post-upgrade` within the runtime environment of your Nautobot installation after updating the `nautobot-ssot` package via `pip`.
+
+### Potential Apps Conflicts
+
+!!! warning
+    If upgrading from `1.x` version to `2.x` version of `nautobot-ssot` app, note that it now incorporates features previously provided by individual apps.
```

Comment:
```suggestion
    If upgrading from versions prior to 1.4 of the `nautobot-ssot` app, note that it now incorporates features previously provided by individual apps.
```

Reply:
We're going to make 1.4.0 the version for all integrations and 2.x is going to be Nautobot 2.0 compatibility.

Reply:
Fixed